Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -472,7 +505,7 @@ private async Task ProcessMessage(CancellationToken cancellationToken)
                 }
             }
 
-            _logger.Debug($"{nameof(ProcessMessage)} is ended. Connected: {_control.Connected}");
+            _logger.Debug($"{nameof(ProcessMessage)} is ended. Connected: {_control!.Connected}");
```

Comment:
`!` is ok?

Reply:
This method is invoked after the _control field is created, so the `!` is safe